access-checkout/src/main/java/com/worldpay/access/checkout/client/session/model/CardDetails.kt

```diff
@@ -13,22 +15,42 @@ class CardDetails private constructor(
     val cvc: String?
 ) {
 
+    /**
+     * This build helps building the [CardDetails] instance
+     */
     data class Builder(
         private var pan: String? = null,
         private var expiryDate: ExpiryDate? = null,
         private var cvc: String? = null
     ) {
 
+        /**
+         * Sets the pan number for the card
+         *
+         * @param[pan] [String] that represents the pan number
+         */
         fun pan(pan: String) = apply { this.pan = pan }
 
-        fun expiryDate(expiryDate: String) = apply {
-            this.expiryDate = ExpiryDate(expiryDate)
-        }
-
+        /**
+         * Sets the expiry date for the card
+         *
+         * @param[expiryDate] [String] that represents the expiry date
```

Comment:
Do we want to also include the expected format? Thoughts?

Reply:
I don't mind to be honest, just feel like if we update the format in the future, we will forget to update the documentation. As most people do.

Reply:
Agreed